src/tasks/onetime/AutoCombatLogic.py

```
@@ -70,6 +102,135 @@ def _periodic_search(self):
+        Returns:
+            tuple[signal, had_action]:
+                signal —— "" 正常 / "break" / "return_false"（来自 normal_ 内嵌循环）。
+                had_action —— 本帧是否产出了条件动作（供立即释放判断）。
+        """
+        # 战技重试：上一帧 digit token 因技力不足失败，本帧重试同一 token
+        if self._pending_skill_token is not None:
+            token = self._pending_skill_token
+            self._pending_skill_token = None
+            success, signal = self._exec_rotation_token(token, deadline)
+            if not success and signal == "":
+                # 仍然技力不足，继续暂存等待下帧
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)